app/controllers/account/admin/billing_contracts_imports_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Account::Admin::BillingContractsImportsController < Account::Admin::AdminController
+  def new
+    @billing_contracts_import = BillingContractsImport.new
+  end
+
+  def create
```

Comment:
all this actions must me in admin/billing_contracts_controller.rb

with names 
def new_import
def import